### PR TITLE
Update nokogiri to 1.10.4

### PR DESCRIPTION
**Pull request by dasanchez**
Update Gemfile.lock to address published vulnerability.

### CODE REVIEW

**Comment by dasanchez** [comment]
@darkdrgn2k @Shrinks99 
The IPFS link to the archived talks from Our Networks 2018 seems to be broken. Do you mind if I remove it?
https://ipfs.io/ipfs/QmQQfD1ztg1aG82VmUmutusbxn7tL4c2o1qHv1ivyWbNj6/

It's in the Timeline page (July 13, 2018).

**Comment by Shrinks99** [comment]
Lets change that link to https://2018.ournetworks.ca/conference/recorded-talks/ instead of removing and then approved!

**Comment by dasanchez** [comment]
Travis is complaining about links on the Build a Node page- it's just the image to the nightly Pine builds.

**Review by Shrinks99 — APPROVED** [review]

File: _timeline/2018-07-13-state-of-our-networks-2018.md [hunk 1]
```diff
@@ -2,4 +2,4 @@
 title: State of Our Networks 2018
 date: 2018-07-13
 ---
-Members helped organize the second [State of Our Networks](https://ournetworks.ca/2018) in Toronto, with the theme of _Beyond DIY: Do It With Others_. Toronto Mesh members [streamed the conference over IPFS](https://github.com/tomeshnet/ipfs-live-streaming). [Recorded videos](https://ournetworks.ca/2018) of the talks archived on [IPFS](https://ipfs.io/ipfs/QmQQfD1ztg1aG82VmUmutusbxn7tL4c2o1qHv1ivyWbNj6/) and [YouTube](https://www.youtube.com/playlist?list=PLx7_J32Ys60ey2bgSn2soAoBy0v9bBUbT).
+Members helped organize the second [State of Our Networks](https://ournetworks.ca/2018) in Toronto, with the theme of _Beyond DIY: Do It With Others_. Toronto Mesh members [streamed the conference over IPFS](https://github.com/tomeshnet/ipfs-live-streaming). [Recorded videos](https://ournetworks.ca/2018) of the talks archived on [IPFS](https://2018.ournetworks.ca/conference/recorded-talks/) and [YouTube](https://www.youtube.com/playlist?list=PLx7_J32Ys60ey2bgSn2soAoBy0v9bBUbT).
```
Comment (benhylau) [inline _timeline/2018-07-13-state-of-our-networks-2018.md]:
```suggestion
Members helped organize the second [State of Our Networks](https://2018.ournetworks.ca) in Toronto, with the theme of _Beyond DIY: Do It With Others_. Toronto Mesh members [streamed the conference over IPFS](https://github.com/tomeshnet/ipfs-live-streaming). [Recorded videos of the talks](https://2018.ournetworks.ca/conference/recorded-talks/) are archived on IPFS, Internet Archive, and YouTube.
```

**Comment by benhylau** [comment]
I made an edit to Our Networks links. Can we merge that and try travis again after we merge the pine64 patch I opened on prototype?

**Pull request merged**